TWOCLIENTTEST.md

```diff
@@ -0,0 +1,111 @@
+# Codex Two-Client Test
+
+The two-client test is a manual test you can perform to check your setup and familiarize yourself with the Codex API. These steps will guide you through running and connecting two nodes, in order to upload a file to one and then download that file from the other. For the purpose of this test we will be running Codex disconnected from any Ethereum nodes, so no currency is required. Additionally, the contracts/sales/marketplace APIs will be unavailable for this reason.
+
+## Prerequisite
+
+Make sure you have built the client, and can run it as explained in the [README](README.md).
+
+## Steps
+
+### 1. Launch Node #1
+
+Open a terminal and run:
+- Mac/Unx: `"build/codex" --data-dir="$(pwd)\Data1" --listen-addrs=/ip4/127.0.0.1/tcp/8070 --api-port=8080  --disc-port=8090`
+- Windows: `"build/codex.exe" --data-dir="Data1" --listen-addrs=/ip4/127.0.0.1/tcp/8070 --api-port=8080 --disc-port=8090`
+
+(Hint: If your terminal interprets the '/' in the listen-address as a reference to your root path, try running the command from a shell-script!)
```

Comment:
Can you add quotes in Windows to avoid this?

Reply:
Tried that, it doesn't like it. I added a hint for the one way I found to get this to work.